src/java.desktop/share/classes/java/awt/PopupMenu.java

```diff
@@ -43,6 +43,10 @@
  * (e.g., you add it to a {@code MenuBar}), then you <b>cannot</b>
  * call {@code show} on that {@code PopupMenu}.
  *
+ * @implNote On Linux systems using Wayland, the popup menu may not be dismissed
```

Comment:
I guess the XWayland is affected, but implementation of the toolkit for Wayland will not have this issue?

Reply:
It should, and this is another point about not changing the documentation.

Reply:
Then let's not update it.